gapic/api/artman_longrunning.yaml

```diff
@@ -1,5 +1,5 @@
 common:
-  api_name: google-longrunning
+  api_name: google-longrunning-v1
```

Comment:
I guess this change only matters if the Python gRPC code for LRO is shipped as a separate package, rather than as part of `googleapis-common-protos`. However, I'm reluctant to do that since common-protos is already post-1.x.

The GAPIC portion will be merged into `google-gax` (the `gax-python` package on PyPI), so it won't need its own package name.